pkgs/top-level/all-packages.nix

```diff
@@ -11419,121 +11419,123 @@ in
      for a specific kernel.  This function can then be called for
      whatever kernel you're using. */
 
-  linuxPackagesFor = kernel: self: let callPackage = newScope self; in rec {
+  linuxPackagesFor = kernel: lib.makeExtensible (self: {
+    callPackage = newScope self;
+
     inherit kernel;
 
-    accelio = callPackage ../development/libraries/accelio { };
+    accelio = self.callPackage ../development/libraries/accelio { };
 
-    acpi_call = callPackage ../os-specific/linux/acpi-call {};
+    acpi_call = self.callPackage ../os-specific/linux/acpi-call {};
```

Comment:
I would prefer if this line could be read as `super.callPackage`, in order to avoid mistakes while using the overlays.